### ISSUE TITLE
Footer link text difficult to read

### ISSUE THREAD

**Issue by nickcolley** [alphagov/govuk-frontend#1420]
This issue is from a May 2019 external accessibility audit report.

**WCAG Reference**: [1.4.6 Contrast (Enhanced) (Level AAA)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-enhanced.html)
**Issue ID**: DAC_3
**URL**: Throughout

## Summary
There were some instances where colour failed to meet the AAA colour contrast ratio

## Screen Shot
![](https://user-images.githubusercontent.com/2445413/58793087-51f0f500-85ed-11e9-91d2-321c55abddb7.png)

The colours of the text in the footer fails to meet AAA for colour contrast and was difficult for colour blind and low vision users to read.

## Current Code Ref(s)
**Color**: #454A4C
**Background**: #DEE0E2 Ratio: 6.78:1

## Low vision user comments
> “I found it difficult to read the link text in the footer of the website as the colours used were dark grey on grey. Increasing this contrast would help me to read the content.”

## Solution
As far as contrast ratio is concerned, it must at least be:
• if text is not bold and its size is less than 18pt: 7:1 for AAA level;
• if text is not bold and its size is at least 18pt: 4.5:1 for AAA level;
• if text is bold and its size is less than 14pt: 7:1 for AAA level;
• if text is bold and its size is at least 14pt: 4.5:1 for AAA level

**Comment by nickcolley** [comment]
The new colours have a contrast ratio of 17.52 which passes level AAA and addresses this feedback, so going to close this out.

<img width="235" alt="Screen Shot 2019-06-03 at 12 38 01" src="https://user-images.githubusercontent.com/2445413/58799170-76a09900-85fc-11e9-8bb8-435a06263901.png">

**Comment by nickcolley** [comment]
Confirmed as fixed in a re-test by DAC on 7th October 2019
